pkg/resmap/factory_test.go

```diff
@@ -273,7 +270,7 @@ func TestNewResMapFromSecretArgs(t *testing.T) {
 				"metadata": map[string]interface{}{
 					"name": "apple",
 				},
-				"type": string(v1.SecretTypeOpaque),
+				"type": "Opaque",
```

Comment:
Please define a constant in ifc.go and use the named constant here and below (rather than raw string)

// See core.v1.SecretTypeOpaque
const SecretTypeOpaque = "Opaque"